internal/command/v2/preparation/command_test.go

```diff
@@ -141,6 +142,27 @@ func Test_create(t *testing.T) {
 	}
 }
 
+func Test_transactionFilter(t *testing.T) {
+	type args struct {
+		filter   FilterToQueryReducer
+		commands []eventstore.Command
+	}
+	tests := []struct {
+		name string
+		args args
+		want FilterToQueryReducer
+	}{
+		// TODO: Add test cases.
```

Comment:
is it intended to have no test cases?

Reply:
as discussed on Friday, there were some tests and validations missing. I added todos so we don't forget to implement them later on